src/explore-education-statistics-frontend/src/modules/find-statistics/components/ContentSubBlockRenderer.tsx

```diff
@@ -46,6 +53,7 @@ const ContentSubBlockRenderer = ({ block, id, publication }: Props) => {
               </ButtonLink>
             </div>
           }
+          onToggle={onToggle}
```

Comment:
Have we checked this is valid when no onToggle is passed in e.g.  it's undefined?

Reply:
@HiveIT-Lewis